githubmarkdownui/ci.py

```diff
@@ -54,12 +90,54 @@ def ci_task_list(self, status: Optional[CIStatus] = None) -> str:
 
         :param status: Only tasks with the given status will be displayed. If not given, all tasks will be displayed
         """
-        pass
+        # Want to display the results like this, and have the additional info in each line aligned with each other:
```

Comment:
Do you think it's worth calling out in doc string that this function only lists out tasks from the parent job?

Reply:
good idea, done